packages/shared/src/components/cards/AdCard.tsx

```diff
@@ -45,10 +44,10 @@ export const AdCard = forwardRef(function AdCard(
   }, []);
 
   return (
-    <Card {...props} className={classNames(className, styles.ad)} ref={ref}>
+    <Card {...props} className={className} ref={ref}>
```

Comment:
No longer needed as `className` is part of the `{...props}`

```suggestion
    <Card {...props} ref={ref}>
```